bsip-0074.md

```diff
@@ -24,13 +24,89 @@ To ensure the debt positions can always be closed with suffcient smartcoin, it i
+
+Here, `margin_call_order_price` is the price at which margin calls will enter the market order book, and `median_feed_price`<sup>(1)</sup> conveys the "even exchange" price between DEBT and COLLATERAL, as reported by the price feed publishers.  Both prices are expressed in units of DEBT / COLLATERAL.
+
+This new calculation of MCOP differs from previous behavior.  Previously, margin calls entered the book at the Maximum Short Squeeze Price (MSSP), defined as:
+
+`max_short_squeeze_price = median_feed_price / MSSR`.
+
+The MSSP is retained under this BSIP as the collateralization threshold at which global settlement can be triggered, but it no longer determines the order price.
+
+_**Note (1):** In the core source code, as of this writing, what we here call `median_feed_price` is called `settlement_price`, although it represents the median reported price feed.  Even though it would match current code, we have preferred not to use the symbol `settlement_price` as a synonym for median feed price in this document for two reasons: (1) the value is not identical with the price at which force settlements occur, due to the Force Settlement Offset (FSO), thus the symbol is confusing, and (2) [BSIP-0071](bsip-0071.md) uses the symbol `settlement_price` to refer to the price at which force settlements occur, inclusive of the FSO, and we do not wish to add ambiguity by using a different definition here._
+
+### Fee Calculation
+
+#### Review of order-matching sequence:
+
+A margin call will enter the book at the MCOP.  However, if at the time the call enters the book, (or any time thereafter in response to a price feed update), it can be matched with an existing limit order at a more favorable price, it will take that order. This means that the `match_price` will be some price no worse than (from the perspective of the callee) the MCOP:
+
+`match_price >= margin_call_order_price`.  _(when margin call is a taker)_
+
+If no immediate match is found, then the call order will wait on the books (as a maker) for a future limit order to take it.  In this case, the eventual match price is exactly the order price:
+
+`match_price = margin_call_order_price`.  _(when margin call is a maker)_
+
+#### Calculation of fee, general case:
+
+When a call order is matched to a limit order and they exchange _X_ amount of smartcoin, this BSIP proposes that two distinct quantities be calculated: the amount of collateral which the limit order receives from the call order in exchange for _X_, and the total amount of collateral that the call order relinquishes to satisfy obligations including the margin call fee.  If the MCFR parameter is zero, then these two quantities are equal, and the behavior is unchanged from before.  But if the MCFR is non-zero, then the relinquished collateral will be a greater quantity than what is payed to the limit order.  The excess is to be paid to the issuer of _X_ as the _Margin Call Fee_.
```

Comment:
Personally I prefer `owner` over `issuer` when talking about smartcoins, although `issuer` is the variable name used in the code.